account_payment_order/models/account_payment_line.py

```diff
@@ -154,3 +154,18 @@ def draft2open_payment_line_check(self):
             raise UserError(
                 _("Missing Partner Bank Account on payment line %s") % self.name
             )
+
+    def get_requested_date(self, today):
```

Comment:
If it should always be computed with today therefore the today should be in the method and not a parameter.
If you put a parameter to be more generic therefore you should call this parameter date, default_date, ref_date or something more generic than today.